Sources/Datadog/RUMMonitor.swift

```diff
@@ -254,6 +270,13 @@ public class RUMMonitor: DDRUMMonitor {
 
     // MARK: - Private
 
+    private func aggregate(_ localAttributes: [AttributeKey: AttributeValue]?) -> [AttributeKey: AttributeValue] {
+        if let someAttributes = localAttributes {
+            return attributes.merging(someAttributes) { $1 }
+        }
+        return attributes
```

Comment:
We have this extension defined in `RUMScope.swift`:
```swift
extension Dictionary where Key == AttributeKey, Value == AttributeValue {
    /// Merges given `rumCommandAttributes` to current dictionary, by overwriting values.
    mutating func merge(rumCommandAttributes: [AttributeKey: AttributeValue]?) {
        guard let additionalAttributes = rumCommandAttributes else {
            return
        }
        merge(additionalAttributes) { _, new in new }
    }
}
```
It is also covered with tests (to ensure proper merging priority). Maybe we can reuse it here?